src/twisted/newsfragments/10209.bugfix

```diff
@@ -0,0 +1 @@
+twisted.protocols.ftp.FTP.ftp_PASS() now constructs twisted.cred.credentials.UsernamePassword using bytes instead of str. twisted.cred.credentials.IUsernamePassword expects username and password to be bytes. Passing a str caused checker twisted.cred.checkers.FilePasswordDB to consistently fail to authenticate.
```

Comment:
```suggestion
twisted.protocols.ftp.FTP.ftp_PASS() now constructs twisted.cred.credentials.UsernamePassword using bytes instead of str. twisted.cred.credentials.IUsernamePassword expects username and password to be bytes. Passing a str caused checker twisted.cred.checkers.FilePasswordDB to consistently fail to authenticate.
```